Makefile

```diff
@@ -34,13 +31,23 @@ endif
 endif
 
 # Environment
+UNAME_S := $(shell uname -s)
 BASE_PATH := $(shell pwd)
 HUGO_PATH := $(BASE_PATH)/.hugo
 HUGO_NAME := hugo
 WORKDIR := $(PWD)
 BUILD_PATH := $(WORKDIR)/build
 LANDING_ARTIFACT := landing_$(TAG).tar.gz
 
+# System
+ifneq ($(UNAME_S),Linux)
+ifeq ($(UNAME_S),Darwin)
+OS := macOS
+else
+$(error "error Unexpected OS; Only Linux or Darwin supported.")
+endif
+endif
```

Comment:
Yes, Markdown s****, but with indentations, it means another thing, and it fails
#wtf